x-pack/plugins/security_solution/server/lib/risk_engine/risk_engine_data_client.ts

```diff
@@ -282,13 +279,6 @@ export class RiskEngineDataClient {
       };
 
       await Promise.all([
-        createOrUpdateIlmPolicy({
-          logger: this.options.logger,
-          esClient,
-          name: ilmPolicyName,
-          policy: ilmPolicy,
-          dataStreamAdapter: this.options.dataStreamAdapter,
```

Comment:
`dataStreamAdapter` is no longer needed in the data client and can be removed. Similarly, references (and comments) in the task code can also be deleted as part of this.